examples/resources/integration/alertmanager.tf

```diff
@@ -22,6 +24,7 @@ resource "shoreline_integration" "am2" {
   serial_number    = "003"
   service_name     = "alertmanager"
   external_url     = "http://alertmanager-2-external-url.io"
+  payload_paths    = ["$.alertmanager.payload.path.2"]
```

Comment:
Is `$.alertmanager.payload.path.2` valid? Can we put, instead, a valid json path for an alert payload? something like `$.labels.host_name`